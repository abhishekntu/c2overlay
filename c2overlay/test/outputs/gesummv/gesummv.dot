digraph graphname {
N48 [color="black", label="store_Imm_1_N61_N48"];
N41 [color="black", label="N39_mul_N39_N41"];
N40 [color="black", label="N38_mul_Imm_6_N38_N40"];
N43 [color="black", label="N32_mul_N32_N43"];
N42 [color="black", label="N30_mul_N30_N42"];
N45 [color="black", label="N37_mul_N37_N45"];
N44 [color="black", label="N34N22_mul_N34_add_N22_N44"];
N47 [color="black", label="store_Imm_0_N62_N47"];
N46 [color="black", label="store_Imm_2_N63_N46"];
N29 [color="black", label="N44N55_mul_N44_add_N55_N29"];
N28 [color="black", label="N45N54_mul_N45_add_N54_N28"];
N23 [color="black", label="N60N50_mul_N60_add_N50_N23"];
N22 [color="black", label="N48N53N49_add_N48_mul_Imm_11_N53_add_N49_N22"];
N21 [color="black", label="load_Imm_3_N18_N21"];
N20 [color="black", label="load_Imm_0_N15_N20"];
N27 [color="black", label="N42_mul_N42_N27"];
N26 [color="black", label="N43N31_mul_N43_add_N31_N26"];
N25 [color="black", label="N40N51_mul_N40_add_N51_N25"];
N24 [color="black", label="N41_mul_N41_N24"];
N8 [color="black", label="load_Imm_8_N2_N8"];
N9 [color="black", label="load_Imm_0_N3_N9"];
N1 [color="black", label="load_Imm_4_N19_N1"];
N2 [color="black", label="load_Imm_1_N14_N2"];
N3 [color="black", label="load_Imm_5_N21_N3"];
N4 [color="black", label="load_Imm_1_N20_N4"];
N5 [color="black", label="load_Imm_0_N8_N5"];
N6 [color="black", label="load_Imm_4_N9_N6"];
N7 [color="black", label="load_Imm_8_N1_N7"];
N12 [color="black", label="load_Imm_6_N6_N12"];
N13 [color="black", label="load_Imm_1_N7_N13"];
N10 [color="black", label="load_Imm_7_N4_N10"];
N11 [color="black", label="load_Imm_2_N5_N11"];
N16 [color="black", label="load_Imm_5_N10_N16"];
N17 [color="black", label="load_Imm_2_N11_N17"];
N14 [color="black", label="load_Imm_3_N12_N14"];
N15 [color="black", label="load_Imm_2_N13_N15"];
N18 [color="black", label="load_Imm_7_N16_N18"];
N19 [color="black", label="load_Imm_6_N17_N19"];
N38 [color="black", label="N56N35N57_add_N56_mul_Imm_11_N35_add_N57_N38"];
N39 [color="black", label="N52_mul_N52_N39"];
N30 [color="black", label="N47_mul_Imm_6_N47_N30"];
N31 [color="black", label="N46_mul_N46_N31"];
N32 [color="black", label="N29N25_mul_N29_add_N25_N32"];
N33 [color="black", label="N28_mul_Imm_11_N28_N33"];
N34 [color="black", label="N59_mul_N59_N34"];
N35 [color="black", label="N23N33N24_add_N23_mul_Imm_6_N33_add_N24_N35"];
N36 [color="black", label="N27N26_mul_N27_add_N26_N36"];
N37 [color="black", label="N58N36_mul_N58_add_N36_N37"];
N41 -> N25;
N40 -> N22;
N43 -> N35;
N42 -> N36;
N45 -> N44;
N44 -> N35;
N29 -> N30;
N28 -> N29;
N23 -> N22;
N22 -> N47;
N21 -> N41;
N20 -> N26;
N27 -> N26;
N26 -> N37;
N25 -> N38;
N24 -> N38;
N8 -> N43;
N9 -> N44;
N9 -> N31;
N9 -> N34;
N9 -> N26;
N9 -> N42;
N9 -> N41;
N1 -> N25;
N2 -> N27;
N3 -> N24;
N4 -> N45;
N4 -> N25;
N4 -> N28;
N4 -> N23;
N4 -> N27;
N4 -> N36;
N5 -> N34;
N6 -> N28;
N7 -> N32;
N12 -> N42;
N13 -> N23;
N10 -> N36;
N11 -> N39;
N11 -> N29;
N11 -> N32;
N11 -> N24;
N11 -> N43;
N11 -> N37;
N16 -> N29;
N17 -> N39;
N14 -> N31;
N15 -> N37;
N18 -> N45;
N19 -> N44;
N38 -> N48;
N39 -> N22;
N30 -> N38;
N31 -> N28;
N32 -> N33;
N33 -> N35;
N34 -> N23;
N35 -> N46;
N36 -> N32;
N37 -> N40;
}
