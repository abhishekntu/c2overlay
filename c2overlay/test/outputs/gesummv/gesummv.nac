procedure foo()
{
  localvar s32 D_2410;
  localvar s32 D_2411;
  localvar s32 D_2412;
  localvar s32 D_2413;
  localvar s32 D_2414;
  localvar s32 D_2415;
  localvar s32 D_2416;
  localvar s32 D_2417;
  localvar s32 D_2418;
  localvar s32 D_2419;
  localvar s32 D_2420;
  localvar s32 D_2421;
  localvar s32 D_2422;
  localvar s32 D_2423;
  localvar s32 D_2424;
  localvar s32 D_2425;
  localvar s32 D_2426;
  localvar s32 D_2427;
  localvar s32 D_2428;
  localvar s32 D_2429;
  localvar s32 D_2430;
  localvar s32 D_2431;
  localvar s32 D_2432;
  localvar s32 D_2433;
  localvar s32 D_2434;
  localvar s32 D_2435;
  localvar s32 D_2436;
  localvar s32 D_2437;
  localvar s32 D_2438;
  localvar s32 D_2439;
  localvar s32 D_2440;
  localvar s32 D_2441;
  localvar s32 D_2442;
  localvar s32 D_2443;
  localvar s32 D_2444;
  localvar s32 D_2445;
  localvar s32 D_2446;
  localvar s32 D_2447;
  localvar s32 D_2448;
  localvar s32 D_2449;
  localvar s32 D_2450;
  localvar s32 D_2451;
  localvar s32 D_2452;
  localvar s32 D_2453;
  localvar s32 D_2454;
  localvar s32 D_2455;
  localvar s32 D_2456;
  localvar s32 D_2457;
  localvar s32 D_2458;
  localvar s32 D_2459;
  localvar s32 D_2460;
  localvar s32 D_2461;
  localvar s32 D_2462;
  localvar s32 D_2463;
  localvar s32 D_2464;
  localvar s32 D_2465;
  localvar s32 D_2466;
  localvar s32 D_2467;
  localvar s32 D_2468;
  localvar s32 D_2469;
  localvar s32 A[9];
  localvar s32 B[9];
  localvar s32 x[3];
  localvar s32 y[3];
  localvar s32 alpha;
  localvar s32 beta;
  localvar s32 i;
  localvar s32 j;
  localvar s32 t00000;
  localvar s32 t00001;
  localvar s32 t00002;
  localvar s32 t00003;
  localvar s32 t00004;
  localvar s32 t00005;
  localvar s32 t00006;
  localvar s32 t00007;
  localvar s32 t00008;
  localvar s32 t00009;
  localvar s32 t00010;
  localvar s32 t00011;
  localvar s32 t00012;
  localvar s32 t00013;
  localvar s32 t00014;
  localvar s32 t00015;
  localvar s32 t00016;
  localvar s32 t00017;
  localvar s32 t00018;
  localvar s32 t00019;
  localvar s32 t00020;
  localvar s32 t00021;
  localvar s32 t00022;
  localvar s32 t00023;
  localvar s32 t00024;
  localvar s32 t00025;
  localvar s32 t00026;
  localvar s32 t00027;
  localvar s32 t00028;
  localvar s32 t00029;
  localvar s32 t00030;
  localvar s32 t00031;
  localvar s32 t00032;
  localvar s32 t00033;
  localvar s32 t00034;
  localvar s32 t00035;
  localvar s32 t00036;
  localvar s32 t00037;
  localvar s32 t00038;
L0005:
  alpha <= ldc 6;
  beta <= ldc 11;
  D_2410 <= load A,0;
  D_2411 <= load x,0;
  D_2412 <= mul D_2410,D_2411;
  D_2413 <= load A,1;
  D_2414 <= load x,1;
  D_2415 <= mul D_2413,D_2414;
  D_2416 <= add D_2412,D_2415;
  D_2417 <= load A,2;
  D_2418 <= load x,2;
  D_2419 <= mul D_2417,D_2418;
  D_2420 <= add D_2416,D_2419;
  D_2421 <= mul D_2420,alpha;
  D_2422 <= load B,0;
  D_2411 <= load x,0;
  D_2423 <= mul D_2422,D_2411;
  D_2424 <= load B,1;
  D_2414 <= load x,1;
  D_2425 <= mul D_2424,D_2414;
  D_2426 <= add D_2423,D_2425;
  D_2427 <= load B,2;
  D_2418 <= load x,2;
  D_2428 <= mul D_2427,D_2418;
  D_2429 <= add D_2426,D_2428;
  D_2430 <= mul D_2429,beta;
  D_2431 <= add D_2421,D_2430;
  y <= store D_2431,0;
  D_2432 <= load A,3;
  D_2411 <= load x,0;
  D_2433 <= mul D_2432,D_2411;
  D_2434 <= load A,4;
  D_2414 <= load x,1;
  D_2435 <= mul D_2434,D_2414;
  D_2436 <= add D_2433,D_2435;
  D_2437 <= load A,5;
  D_2418 <= load x,2;
  D_2438 <= mul D_2437,D_2418;
  D_2439 <= add D_2436,D_2438;
  D_2440 <= mul D_2439,alpha;
  D_2441 <= load B,3;
  D_2411 <= load x,0;
  D_2442 <= mul D_2441,D_2411;
  D_2443 <= load B,4;
  D_2414 <= load x,1;
  D_2444 <= mul D_2443,D_2414;
  D_2445 <= add D_2442,D_2444;
  D_2446 <= load B,5;
  D_2418 <= load x,2;
  D_2447 <= mul D_2446,D_2418;
  D_2448 <= add D_2445,D_2447;
  D_2449 <= mul D_2448,beta;
  D_2450 <= add D_2440,D_2449;
  y <= store D_2450,1;
  D_2451 <= load A,6;
  D_2411 <= load x,0;
  D_2452 <= mul D_2451,D_2411;
  D_2453 <= load A,7;
  D_2414 <= load x,1;
  D_2454 <= mul D_2453,D_2414;
  D_2455 <= add D_2452,D_2454;
  D_2456 <= load A,8;
  D_2418 <= load x,2;
  D_2457 <= mul D_2456,D_2418;
  D_2458 <= add D_2455,D_2457;
  D_2459 <= mul D_2458,alpha;
  D_2460 <= load B,6;
  D_2411 <= load x,0;
  D_2461 <= mul D_2460,D_2411;
  D_2462 <= load B,7;
  D_2414 <= load x,1;
  D_2463 <= mul D_2462,D_2414;
  D_2464 <= add D_2461,D_2463;
  D_2465 <= load B,8;
  D_2418 <= load x,2;
  D_2466 <= mul D_2465,D_2418;
  D_2467 <= add D_2464,D_2466;
  D_2468 <= mul D_2467,beta;
  D_2469 <= add D_2459,D_2468;
  y <= store D_2469,2;
  L_EXIT <= jmpun;
L_EXIT:
  nop;
}
