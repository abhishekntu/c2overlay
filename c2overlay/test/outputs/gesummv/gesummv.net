.input N21
pinlist: N21

.input N20
pinlist: N20

.input N8
pinlist: N8

.input N9
pinlist: N9

.input N1
pinlist: N1

.input N2
pinlist: N2

.input N3
pinlist: N3

.input N4
pinlist: N4

.input N5
pinlist: N5

.input N6
pinlist: N6

.input N7
pinlist: N7

.input N12
pinlist: N12

.input N13
pinlist: N13

.input N10
pinlist: N10

.input N11
pinlist: N11

.input N16
pinlist: N16

.input N17
pinlist: N17

.input N14
pinlist: N14

.input N15
pinlist: N15

.input N18
pinlist: N18

.input N19
pinlist: N19

.output out:N48
pinlist: N48

.output out:N47
pinlist: N47

.output out:N46
pinlist: N46

.clb N41_blk
pinlist: N9 N21 open open N41 open open open open 
subblock: N41_blk 0 1 open open 4 open open open open

.clb N40_blk
pinlist: N37 open open open N40 open open open open 
subblock: N40_blk 0 open open open 4 open open open open

.clb N43_blk
pinlist: N8 N11 open open N43 open open open open 
subblock: N43_blk 0 1 open open 4 open open open open

.clb N42_blk
pinlist: N9 N12 open open N42 open open open open 
subblock: N42_blk 0 1 open open 4 open open open open

.clb N45_blk
pinlist: N4 N18 open open N45 open open open open 
subblock: N45_blk 0 1 open open 4 open open open open

.clb N44_blk
pinlist: N9 N19 N45 open N44 open open open open 
subblock: N44_blk 0 1 2 open 4 open open open open

.clb N29_blk
pinlist: N28 N11 N16 open N29 open open open open 
subblock: N29_blk 0 1 2 open 4 open open open open

.clb N28_blk
pinlist: N31 N4 N6 open N28 open open open open 
subblock: N28_blk 0 1 2 open 4 open open open open

.clb N23_blk
pinlist: N34 N4 N13 open N23 open open open open 
subblock: N23_blk 0 1 2 open 4 open open open open

.clb N22_blk
pinlist: N23 N39 N40 open N47 open open open open 
subblock: N22_blk 0 1 2 open 4 open open open open

.clb N27_blk
pinlist: N2 N4 open open N27 open open open open 
subblock: N27_blk 0 1 open open 4 open open open open

.clb N26_blk
pinlist: N27 N9 N20 open N26 open open open open 
subblock: N26_blk 0 1 2 open 4 open open open open

.clb N25_blk
pinlist: N1 N4 N41 open N25 open open open open 
subblock: N25_blk 0 1 2 open 4 open open open open

.clb N24_blk
pinlist: N3 N11 open open N24 open open open open 
subblock: N24_blk 0 1 open open 4 open open open open

.clb N38_blk
pinlist: N24 N25 N30 open N48 open open open open 
subblock: N38_blk 0 1 2 open 4 open open open open

.clb N39_blk
pinlist: N11 N17 open open N39 open open open open 
subblock: N39_blk 0 1 open open 4 open open open open

.clb N30_blk
pinlist: N29 open open open N30 open open open open 
subblock: N30_blk 0 open open open 4 open open open open

.clb N31_blk
pinlist: N9 N14 open open N31 open open open open 
subblock: N31_blk 0 1 open open 4 open open open open

.clb N32_blk
pinlist: N36 N7 N11 open N32 open open open open 
subblock: N32_blk 0 1 2 open 4 open open open open

.clb N33_blk
pinlist: N32 open open open N33 open open open open 
subblock: N33_blk 0 open open open 4 open open open open

.clb N34_blk
pinlist: N5 N9 open open N34 open open open open 
subblock: N34_blk 0 1 open open 4 open open open open

.clb N35_blk
pinlist: N33 N43 N44 open N46 open open open open 
subblock: N35_blk 0 1 2 open 4 open open open open

.clb N36_blk
pinlist: N4 N10 N42 open N36 open open open open 
subblock: N36_blk 0 1 2 open 4 open open open open

.clb N37_blk
pinlist: N26 N11 N15 open N37 open open open open 
subblock: N37_blk 0 1 2 open 4 open open open open

