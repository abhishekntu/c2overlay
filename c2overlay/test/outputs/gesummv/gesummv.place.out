Netlist file: test.net   Architecture file: test.arch.xml
Array size: 7 x 7 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N21		8	5	0	#0
N20		1	0	0	#1
N8		4	0	0	#2
N9		8	4	0	#3
N1		8	3	0	#4
N2		2	0	0	#5
N3		5	8	0	#6
N4		8	1	0	#7
N5		3	8	0	#8
N6		0	2	0	#9
N7		8	2	0	#10
N12		3	0	0	#11
N13		0	5	0	#12
N10		6	0	0	#13
N11		0	1	0	#14
N16		4	8	0	#15
N17		0	3	0	#16
N14		0	4	0	#17
N15		0	6	0	#18
N18		8	6	0	#19
N19		7	0	0	#20
out:N48		6	8	0	#21
out:N47		2	8	0	#22
out:N46		5	0	0	#23
N41_blk		7	4	0	#24
N40_blk		2	2	0	#25
N43_blk		4	1	0	#26
N42_blk		3	1	0	#27
N45_blk		7	2	0	#28
N44_blk		7	1	0	#29
N29_blk		4	2	0	#30
N28_blk		3	2	0	#31
N23_blk		2	4	0	#32
N22_blk		2	3	0	#33
N27_blk		2	1	0	#34
N26_blk		1	1	0	#35
N25_blk		7	3	0	#36
N24_blk		5	3	0	#37
N38_blk		6	3	0	#38
N39_blk		1	3	0	#39
N30_blk		4	3	0	#40
N31_blk		3	3	0	#41
N32_blk		6	2	0	#42
N33_blk		5	2	0	#43
N34_blk		3	4	0	#44
N35_blk		5	1	0	#45
N36_blk		6	1	0	#46
N37_blk		1	2	0	#47
